v22.1/dbeaver.md

```diff
@@ -30,43 +30,52 @@ To work through this tutorial, take the following steps:
 
-<img src="{{ 'images/v2.1/dbeaver-01-select-cockroachdb.png' | relative_url }}" alt="DBeaver - Select CockroachDB" style="border:1px solid #eee;max-width:100%" />
+    <img src="{{ 'images/v22.1/dbeaver-01-select-cockroachdb.png' | relative_url }}" alt="DBeaver - Select CockroachDB" style="border:1px solid #eee;max-width:100%" />
 
-## Step 2. Update the connection settings
+1. Click **Next**. The **Connection Settings** dialog displays.
 
-On the **Create new connection** dialog that appears, click **Network settings**.
+1. In the **Database** field, enter **movr**.
 
-<img src="{{ 'images/v2.1/dbeaver-02-cockroachdb-connection-settings.png' | relative_url }}" alt="DBeaver - CockroachDB connection settings" style="border:1px solid #eee;max-width:100%" />
+    <img src="{{ 'images/v22.1/dbeaver-02-cockroachdb-connection-settings.png' | relative_url }}" alt="DBeaver - CockroachDB connection settings" style="border:1px solid #eee;max-width:100%" />
 
-From the network settings, click the **SSL** tab.  It will look like the screenshot below.
+## Step 2. Update the connection settings
 
-<img src="{{ 'images/v2.1/dbeaver-03-ssl-tab.png' | relative_url }}" alt="DBeaver - SSL tab" style="border:1px solid #eee;max-width:100%" />
+1. Click the **SSL** tab.
 
-Check the **Use SSL** checkbox as shown, and fill in the text areas as follows:
+    <img src="{{ 'images/v22.1/dbeaver-03-ssl-tab.png' | relative_url }}" alt="DBeaver - SSL tab" style="border:1px solid #eee;max-width:100%" />
 
-- **Root certificate**: Use the `ca.crt` file you generated for your secure cluster.
-- **SSL certificate**: Use a client certificate generated from your cluster's root certificate.  For the root user, this will be named `client.root.crt`.  For additional security, you may want to create a new database user and client certificate just for use with DBeaver.
-- **SSL certificate key**: Because DBeaver is a Java application, you will need to transform your key file to the `*.pk8` format using an [OpenSSL command](https://wiki.openssl.org/index.php/Command_Line_Utilities#pkcs8_.2F_pkcs5) like the one shown below.  Once you have created the file, enter its location here.  In this example, the filename is `client.root.pk8`.
-    {% include_cached copy-clipboard.html %}
-    ~~~ console
-    $ openssl pkcs8 -topk8 -inform PEM -outform DER -in client.root.key -out client.root.pk8 -nocrypt
-    ~~~
+1. Check the **Use SSL** checkbox as shown, and fill in the text areas as follows:
+    - **Root certificate**: Use the `ca.crt` file you generated for your secure cluster.
+    - **SSL certificate**: Use a client certificate generated from your cluster's root certificate.  For the root user, this will be named `client.root.crt`.  For additional security, you may want to create a new database user and client certificate just for use with DBeaver.
```

Comment:
```suggestion
    - **SSL certificate**: Use a client certificate generated from your cluster's root certificate. For the root user, this will be named `client.root.crt`. For additional security, you may want to create a new database user and client certificate just for use with DBeaver.
```